vitest.examples.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from "vitest/config";
+import tsConfigPaths from "vite-tsconfig-paths";
+import packageJson from "./package.json";
+
+export default defineConfig({
+  test: {
+    globals: true,
+    passWithNoTests: true,
+    testTimeout: 120 * 1000,
+    printConsoleTrace: true,
+    setupFiles: ["./tests/setup.examples.ts"],
```

Comment:
Use the same setup as the E2E tests.

Reply:
or ideally just import `e2e` test config and spread it here

```
export define defineConfig({
   ...e2eConfig,
   // changes
})
```

Reply:
I'm not sure how to do it.  What necessary is undefine `afterEach` function defined in the "setup.ts".